api/types.go

```diff
@@ -221,6 +221,8 @@ type DeleteRequest struct {
 type ShowRequest struct {
 	Model    string `json:"model"`
 	System   string `json:"system"`
+
+	// Template is deprecated
```

Comment:
did you mean to modify this for `/api/show`?

Reply:
Yeah, because before you would send the template to the endpoint and it would substitute it for the one in the model (which was admittedly super weird).